Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/CatrobatImageIOIntegrationTest.kt

```diff
@@ -82,6 +84,14 @@ class CatrobatImageIOIntegrationTest {
                 delete()
```

Comment:
Looks like the file deletion is redundant now. Please check if this is still needed, otherwise clean dead code.

Reply:
removed the redundant deletion in the tearDown()